DirectEmployers/MyJobs
# Update dockerdev.sh

## What's this PR do?
Adds `initdynamodbdata` to `init`
## Screenshots (if appropriate)
N/A
## Where should the reviewer start?
Ensure I wrote the name right, I guess?
## How should this be manually tested?
First, you need a bare computer. Then you follow our setup guide and ensure that it works.
## Any background context you want to provide?
N/A
## PR Checklist

### Front-End


### Separation of Concerns


### Tests


### Migrations
